package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "egeria-api-mocks",
+  "version": "0.0.0",
+  "description": "Egeria API Mocks",
+  "main": "index.js",
+  "scripts": {
+    "start": "node .",
+    "start:dev": "nodemon ."
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/egeria/egeria-api-mocks.git"
+  },
+  "keywords": [
+    "egeria",
+    "api",
+    "mocks"
+  ],
+  "author": "Sirbu Nicolae-Cezar",
```

Comment:
Our egeria repos I think just use Egeria contributors as the author rather than named individuals
For example we have a copyright statement `<!-- Copyright Contributors to the ODPi Egeria project. -->`